### PR TITLE
Add entrypoint for OQC: oqc.remote

### PR DESCRIPTION
**Pull request by rmoyard**
**Context:**
The oqc device was not added to the entry points.

**Description of the Change:**
Add the oqc device to the entry points.\

**Benefits:**
`qml.device('oqc.remote', backend="lucy", wires=2)`

### CODE REVIEW

**Comment by rmoyard** [comment]
[sc-61990]

File: doc/changelog.md [hunk 1]
```diff
@@ -61,15 +61,14 @@
...
-  from catalyst.oqc import OQCDevice
 
   import os
 
   os.environ["OQC_EMAIL"] = "your_email"
   os.environ["OQC_PASSWORD"] = "your_password"
   os.environ["OQC_URL"] = "oqc_url"
 
-  device = OQCDevice(backend="lucy", shots=2012, wires=2)
+  device = qml.device("oqc.remote", backend="lucy", shots=2012, wires=2)
```
Comment (josh146) [inline doc/changelog.md:71]: Only suggestion @rmoyard is to update this to `oqc.cloud` to match the name of their cloud service https://oqc.tech/access/oqc-cloud/

**Review by josh146 — COMMENTED** [review]
Thanks @rmoyard! Will this PR need to be made into the RC branch?

File: doc/changelog.md (hunk 1 above)
Comment (rmoyard) [reply doc/changelog.md:71]: Ok for sure :+1:

**Review by josh146 — APPROVED** [review]
thanks @rmoyard!

**Pull request merged**